patch/common/usart.yaml

```diff
@@ -22,7 +22,6 @@ USART?:
       _replace_enum:
         USART_ASYNC: [0, "Asynchronous USART"]
         USART_SYNC:  [1, "Synchronous USART"]
-        SPI_MASTER:  [3, "Master SPI (MSPIM)"]
```

Comment:
What's this about?

Reply:
I had grouped some duplicated definitions for the UMSEL without SPI_MASTER here, and copied the definition with SPI_MASTER to its own file, but this change has been removed in the stripped down version.